### PR TITLE
Typography component

### PR DESCRIPTION
**Pull request by prath-dev**
Typography Component Added .
Live Preview : http://127.0.0.1:5501/index.html#typography

### CODE REVIEW

**Review by muskan-gupta-0803 — APPROVED** [review]
Looks good to merge.

File: components/Typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+.text-container {
+    text-align: center;
+    height: 19rem;
+    width: 30rem;
+    border: 2px solid #323232;
+}
+
+.Texts-container {
```
Comment (vishalg8454) [inline components/Typography/typography.css:64]: The name of class is capital letter?

File: components/Typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+.medium-text {
+    font-size: 2rem;
+}
+
+.large-text {
+    font-size: 2.5rem;
+}
+
+.strike-Text {
```
Comment (vishalg8454) [inline components/Typography/typography.css:83]: Naming of the class and the class seems not to be doing what is says.

**Review by vishalg8454 — COMMENTED** [review]
Few changes and rest is fine.

File: components/Typography/typography.css (hunk 1 above)
Comment (prath-dev) [reply components/Typography/typography.css:64]: okkk i will look into it

File: components/Typography/typography.css (hunk 2 above)
Comment (prath-dev) [reply components/Typography/typography.css:83]: okkk noted

File: components/Typography/typography.css (hunk 1 above)
Comment (prath-dev) [reply components/Typography/typography.css:64]: done

**Pull request merged**